yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -586,6 +586,7 @@ func (yb *TargetYugabyteDB) ExecuteBatch(migrationUUID uuid.UUID, batch *EventBa
 				return false, fmt.Errorf("failed to update table stats on target db via query-%s: %w, rowsAffected: %v",
 					updateTableStatsQuery, err, res.RowsAffected())
 			}
+			//TODO: confirm do we really need to keep this
```

Comment:
Why do you think that this is not needed?

Reply:
I believe this was done as a workaround earlier for the empty tables as the metadata table was not initialized and rows affected were 0 because of which we were inserting the entry for that table but now metadata tables are also getting initialized for the empty table so these code of line no more required I believe. cc: @makalaaneesh

Reply:
No need to remove this as this is not an issue as such it's just some extra code and may help us in some unknown scenario where the metadata table for a table is not initialized but streaming has events for that, so it's better to keep it.